scripts/monitoring/delete-stuck-projects.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This is a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1367432
+
+# Look for any projects found during the last run of this script
+if [ -f /tmp/stuck_projects ]; then
+  projects_to_delete="$(cat /tmp/stuck_projects)"
+
+  # If the project still exists, is empty, and contains nothing, delete it.
+  for project in ${projects_to_delete}; do
+    oc get project ${project} &> /dev/null
+
+    if [ $? == 0 ] && [ -z "$(oc get all -n ${project})" ]; then
```

Comment:
Because `oc get all -n $project` will return 0 regardless if a project exists or not. It just tells me if the project is empty. And apparently a non-existent project can be empty.

```
[root@dev-preview-stg-master-defb2 ~]# oc get all -n default &> /dev/null; echo $?
0
[root@dev-preview-stg-master-defb2 ~]# oc get all -n this_doesnt_exist &> /dev/null; echo $?
0
```